spec/calendar.html

```diff
@@ -517,6 +517,7 @@ <h1>
         </dd>
       </dl>
       <emu-alg>
+        1. Assert: No non-String primitive value, when coerced to a String, is a built-in calendar identifier.
```

Comment:
I'm not sure we can make this assertion. `false`, `true`, `null`, and 8-digit numbers such as `11111111` are all syntactically legal calendar identifiers. We don't control whether they might become built-in to CLDR in the future, though I agree it's quite unlikely. If we want to be able to assert this, I think we'll need to special-case these identifiers in AvailableCalendars to make sure they are never "available" even if they exist in CLDR.

Reply:
OK, in the latest push I added an additional assertion to AvailableCalendars. Do you think I should also remove the assertion from here?

Reply:
I'll remove it from here because AvailableCalendars is a better home.

Reply:
Actually, after thinking about it more, I don't think we care, because we're outlawing non-string primitives. Even if CLDR adds a calendar called "undefined"... the user still will have to pass a string "undefined" not an `undefined` value. So I'll remove the assertion from both places.